chanjo/store/models.py

```diff
@@ -56,8 +55,7 @@ class Sample(BASE):
     name = Column(types.String(128))
     group_name = Column(types.String(128))
 
-    sample = orm.relationship('TranscriptStat', cascade='all,delete',
-                              backref='sample')
+    stats = orm.relationship('TranscriptStat', cascade='all,delete,delete-orphan')
```

Comment:
Well found!

Reply:
without that param if you removed a sample it didin't remove the linked records in TranscriptStat any more. I guess something has changed in SQLAlchemy when switching to version 2?